askstylo/superset
# add feature: clear filters when active tab changes

### SUMMARY
add feature for clearing filters when changing active tabs

### TESTING INSTRUCTIONS
Create dashboard that uses a drilldown-enabled pie or bar chart inside of a tab and has a second tab containing arbitrary content.  Add 'CLEAR_FILTER_ON_TAB_NAV': True to superset_config.py.  Click to go down one level in the drilldown, then change tabs and change back.  Should observe drilldown chart resetting to top level.

### ADDITIONAL INFORMATION
- [ x ] Required feature flags: CLEAR_FILTER_ON_TAB_NAV, DASHBOARD_DRILL_DOWN
- [ x ] Changes UI
- [ x ] Introduces new feature or API
